openfe/utils/visualization_3D.py

```diff
@@ -107,6 +107,45 @@ def _add_spheres(view:py3Dmol.view, mol1:Chem.Mol, mol2:Chem.Mol, mapping:Dict[i
         )
 
 
+@requires_package("py3Dmol")
+def show_component_coords(mols: Iterable[ExplicitMoleculeComponent],
```

Comment:
"show coordinates" sounds like `print(component.coordinates)`.  How about `visualise_components_3d` or something? `view_components_3d`?

Reply:
I choose: view_components_3d :)

Reply:
this should be changed in show_3D_mapping as well ;)

Reply:
to view_mapping_3d